README.md

```diff
@@ -58,13 +58,13 @@ Haystack is built in a modular fashion so that you can combine the best technolo
-| :ledger: [Docs](https://docs.haystack.deepset.ai)                                   | Components, Pipeline Nodes, Guides, API Reference                                                                                                                                                                                                   |
+| :ledger: [Docs](https://docs.haystack.deepset.ai)                                             | Components, Pipeline Nodes, Guides, API Reference                                                                                                                                                                                                 |
 | :floppy_disk: [Installation](https://github.com/deepset-ai/haystack#floppy_disk-installation) | How to install Haystack                                                                                                                                                                                                                           |
 | :mortar_board: [Tutorials](https://github.com/deepset-ai/haystack#mortar_board-tutorials)     | See what Haystack can do with our Notebooks & Scripts                                                                                                                                                                                             |
 | :beginner: [Quick Demo](https://github.com/deepset-ai/haystack#beginner-quick-demo)           | Deploy a Haystack application with Docker Compose and a REST API                                                                                                                                                                                  |
 | :vulcan_salute: [Community](https://github.com/deepset-ai/haystack#vulcan_salute-community)   | [Discord](https://haystack.deepset.ai/community/join), [Twitter](https://twitter.com/deepset_ai), [Stack Overflow](https://stackoverflow.com/questions/tagged/haystack), [GitHub Discussions](https://github.com/deepset-ai/haystack/discussions) |
 | :heart: [Contributing](https://github.com/deepset-ai/haystack#heart-contributing)             | We welcome all contributions!                                                                                                                                                                                                                     |
-| :bar_chart: [Benchmarks](https://haystack.deepset.ai/benchmarks/)                       | Speed & Accuracy of Retriever, Readers and DocumentStores                                                                                                                                                                                         |
```

Comment:
Are these whitespace changes intentional? Just wondering if they'll break the layout